src/connectors/piped.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+Connector.playerSelector = '#app > div';
+
+Connector.artistTrackSelector =
+	'div.w-full > div:nth-child(1) > div.font-bold.mt-2.text-2xl.break-words';
+
+Connector.getArtistTrack = () => {
+	const text = Util.getTextFromSelectors(Connector.artistTrackSelector);
+	return Util.processYtVideoTitle(text);
+};
+
+Connector.getCurrentTime = () =>
+	document.querySelector('.shaka-video').currentTime;
```

Comment:
This could be extracted into a common variable to avoid duplication and improve refactoring later - but it's not a deal breaker breaker :)